besu/src/main/java/org/hyperledger/besu/cli/options/unstable/EvmOptions.java

```diff
@@ -65,7 +66,10 @@ public static EvmOptions create() {
   @Override
   public EvmConfiguration toDomainObject() {
     return new EvmConfiguration(
-        jumpDestCacheWeightKilobytes, Optional.ofNullable(kzgTrustedSetupFilePath));
+        jumpDestCacheWeightKilobytes,
+        kzgTrustedSetupFilePath != null
+            ? Optional.ofNullable(Path.of(kzgTrustedSetupFilePath))
```

Comment:
nit: result of Path::of is never null

Reply:
This is not a base level configuration option.  This is the wrong place in code to do it.  It should be an argument in the MainnetEVMs methods for relevant versions.